aci/resource_aci_fvrsprovcons.go

```diff
@@ -206,6 +218,7 @@ func resourceAciContractProviderCreate(ctx context.Context, d *schema.ResourceDa
 	contractType := d.Get("contract_type").(string)
 
 	ApplicationEPGDn := d.Get("application_epg_dn").(string)
+	MatchT := d.Get("match_t").(string)
 
 	if contractType == "provider" {
```

Comment:
should we create a function for this attributes set conditional which can be reused by create and update?